src/core.ts

```diff
@@ -602,45 +612,45 @@ export async function kill(pid: number, signal?: string) {
   for (const p of children) {
     try {
       process.kill(+p.pid, signal)
-    } catch (e) {}
+    } catch (e) { }
   }
   try {
     process.kill(-pid, signal)
   } catch (e) {
     try {
       process.kill(+pid, signal)
-    } catch (e) {}
+    } catch (e) { }
   }
 }
 
 export type LogEntry =
   | {
-      kind: 'cmd'
-      verbose: boolean
-      cmd: string
-    }
+    kind: 'cmd'
+    verbose: boolean
+    cmd: string
+  }
   | {
-      kind: 'stdout' | 'stderr'
-      verbose: boolean
-      data: Buffer
-    }
+    kind: 'stdout' | 'stderr'
+    verbose: boolean
+    data: Buffer
+  }
   | {
-      kind: 'cd'
-      dir: string
-    }
+    kind: 'cd'
+    dir: string
+  }
   | {
-      kind: 'fetch'
-      url: RequestInfo
-      init?: RequestInit
-    }
+    kind: 'fetch'
+    url: RequestInfo
+    init?: RequestInit
+  }
   | {
-      kind: 'retry'
-      error: string
-    }
+    kind: 'retry'
+    error: string
+  }
   | {
-      kind: 'custom'
-      data: any
-    }
+    kind: 'custom'
+    data: any
+  }
```

Comment:
I think your prettier settings or something messed here.

Reply:
Let't keep the formatting: `yarn fmt`

Reply:
Thanks @antongolub. I have now fixed the formatting.